### PR TITLE
[VIDECO-7846] Added Experience Composer Insights Sample Code

### PR DESCRIPTION
**Pull request by mbiederman12**
Added the query sample code for Experience Composer's features. Explained the usage and what each returned.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -341,6 +341,34 @@ input `endCursor` parameter to obtain the next page of data. For more informatio
...
 
+### Experience Composer Usage Data
+
+#### Query to Insights API (Sample)
+
+##### This query will return the total usage that `experienceComposer` publishes to a video session (independent of an archiver or broadcast composer), `experienceComposerArchive` publishes to a video session connected to an archiver, and `experienceComposerBroadcast` publishes to a video session connected to a broadcast composer:
+
+```
+{
+  project(projectId: ${apiKey}) {
```
Comment (Raulkg) [inline README.md:352]:
```suggestion
  project(projectId: ${YOUR_API_KEY}) {
```
Let's follow the same pattern as other examples. Makes it easier to understand and follow.

File: README.md [hunk 2]
```diff
@@ -341,6 +341,34 @@ input `endCursor` parameter to obtain the next page of data. For more informatio
...
+      end: ${moment(endDate)},
+      interval: DAILY
+    ) {
+      resources {
+        intervalStart,
+        intervalEnd,
+        usage {
+          experienceComposer,
+          experienceComposerArchive,
+          experienceComposerBroadcast
```
Comment (Raulkg) [inline README.md:364]: units of usage?

Comment (mbiederman12) [reply README.md:364]: Just updated to 'total minutes'

File: README.md [hunk 3]
```diff
@@ -341,6 +341,34 @@ input `endCursor` parameter to obtain the next page of data. For more informatio
 }
 ```
 
+### Experience Composer Usage Data
+
+#### Query to Insights API (Sample)
+
+##### This query will return the daily total minutes that `experienceComposer` publishes to a video session (independent of an archiver or broadcast composer), `experienceComposerArchive` publishes to a video session connected to an archiver, and `experienceComposerBroadcast` publishes to a video session connected to a broadcast composer:
```
Comment (jeffswartz) [inline README.md:348]:
```suggestion
##### This query returns the daily total minutes that [Experience Composer](https://tokbox.com/developer/guides/experience-composer) publishes to a video session (independent of an archiver or broadcast composer), Experience Composer publishes to a video session being archived, and Experience Composer publishes to a video session being sent to a live-streaming broadcast:
```

**Review by jeffswartz — APPROVED** [review]

**Pull request merged**